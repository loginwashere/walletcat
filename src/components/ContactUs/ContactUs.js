import React, { Component } from 'react';

class ContactUs extends Component {
  render() {
    return (
      <h1>Contact Us</h1>
    );
  }
}

export default ContactUs;
