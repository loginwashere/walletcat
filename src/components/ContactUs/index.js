import React, { Component } from 'react';

export class ContactUs extends Component {
  render() {
    return (
      <h1>Contact Us</h1>
    );
  }
}

export default ContactUs;
