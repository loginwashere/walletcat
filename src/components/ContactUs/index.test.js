import React from 'react'
import renderer from 'react-test-renderer'
import ContactUs from '.'

describe('components:ContactUs:', () => {
  it('renders without crashing', () => {
    const tree = renderer.create(
      <ContactUs />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
