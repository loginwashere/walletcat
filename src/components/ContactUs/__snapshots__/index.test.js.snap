exports[`components:ContactUs: renders without crashing 1`] = `
<h1>
  Contact Us
</h1>
`;
